admin/admin_token.go

```diff
@@ -0,0 +1,40 @@
+package admin
+
+import (
+	"time"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/google/uuid"
+	"github.com/vocdoni/blind-csp/types"
+)
+
+// GenerateAdminToken generates a new admin token for an election lasting one month
```

Comment:
Please explain in the comment how do you generate the admin token and how do you ensure it is safe (the entropy source).